pkg/util/util_supported.go

```diff
@@ -43,7 +43,7 @@ func GetRuntimeDir() (string, error) {
 				logrus.Debugf("unable to make temp dir: %v", err)
 			}
 			st, err := os.Stat(tmpDir)
-			if err == nil && int(st.Sys().(*syscall.Stat_t).Uid) == os.Geteuid() && st.Mode().Perm() == 0700 {
+			if err == nil && int(st.Sys().(*syscall.Stat_t).Uid) == os.Geteuid() && st.Mode().Perm() >= 0700 {
```

Comment:
Suggestion: check `.Perm & 0722 == 0700` instead? That: (1) confirms dir is writable by user, (2) is not writable by others

Reply:
I think `& 0766` might be even better

Reply:
Could we turn the two conditions into a function with a comment?  I am certain I will forget the meaning by tomorrow :^)

Reply:
that is a good idea.

Fixed and pushed a new version